### PR TITLE
[feature/83] Create admin dashboard card

### LINKED ISSUE #83: Create the admin dashboard card
## About

- A reusable component 
- It's gonna be used by our dashboard page to render the statistics
- Do not use it yet

image

## Designs

- [figma](https://www.figma.com/file/uLiqrlxmOB5xCppIzmiUXV/Teacher-Workout?node-id=625%3A2474)

### PR DESCRIPTION
**Pull request by kata-kas**
Closes #83 

Admin Dashboard Card with title, icon, statistics, link and linkText as props.
Icons.js maps the prop icon to the corresponding svg icon.

Didn't use a test framework, but I followed the used structure of the other components from the project.

### CODE REVIEW

**Review by radubuciuceanu — CHANGES_REQUESTED** [review]
Let's fix all the linting errors. Can I ask you to run `yarn lint:fix` and solve all the issues, please?

**Comment by kata-kas** [comment]
Ok

**Comment by radubuciuceanu** [comment]
I'll have a second look over it after we fix them.

**Comment by radubuciuceanu** [comment]
Also, for further pull requests, let's not use merging anymore. Let's use rebasing. It helps a lot the reviewer. Ping me if you need any help there and we can easily make a call together.

**Review by radubuciuceanu — CHANGES_REQUESTED** [review]
- Let's move all the new components into the `src/admin/dashboard/card` folder.
- Let's also rollback the `yarn.lock` file.

File: src/admin/admin-dashboard-card/body.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  },
+  statistics: {
+    fontFamily: theme.typography.fontFamily,
+    fontWeight: '600',
+    fontSize: 24,
+    color: '#111827',
+  },
+}))
+
+export const AdminDashboardCardBody = ({ title, statistics }) => (
```
Comment (radubuciuceanu) [inline src/admin/admin-dashboard-card/body.jsx]: Would the name `Body` be more relevant for this component?

Comment (radubuciuceanu) [reply src/admin/admin-dashboard-card/body.jsx]: We'd already have enough context from the folder structure.

File: src/admin/admin-dashboard-card/icon.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+const useStyles = makeStyles(theme => ({
+  iconWrapper: {
+    backgroundColor: '#6366F1',
+    borderRadius: '6px',
+    padding: '12.5%',
+  },
+  icon: { height: '18' },
+}))
+
+export const AdminDashboardCardIcon = ({ icon }) => (
```
Comment (radubuciuceanu) [inline src/admin/admin-dashboard-card/icon.jsx]: Would the name `Icon` be more relevant here?

Comment (radubuciuceanu) [reply src/admin/admin-dashboard-card/icon.jsx]: We'd already have enough context from the folder structure.

File: src/admin/admin-dashboard-card/index.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  makeStyles
+} from '@material-ui/core'
+import PropTypes from 'prop-types'
+import { AdminDashboardCardBody } from 'admin/admin-dashboard-card/body'
+import { AdminDashboardCardIcon } from 'admin/admin-dashboard-card/icon'
+import { AdminDashboardLink } from 'admin/admin-dashboard-card/link'
+
+const useStyles = makeStyles({ cardActions: { backgroundColor: '#F9FAFB' } })
+
+export const AdminDashboardCard = ({ icon, title, statistics, link, linkText }) => (
```
Comment (radubuciuceanu) [inline src/admin/admin-dashboard-card/index.jsx]: Would the name `Card` be more relevant here? We can easily omit the material ui naming collisions by importing it like: `import * as Material from '@material-ui/core'.`

File: src/admin/admin-dashboard-card/link.jsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    fontWeight: 'normal',
+    fontSize: 14,
+    lineHeight: 2,
+    color: '#4F46E5',
+    textDecoration: 'none',
+    paddingLeft: '16',
+  },
+}))
+
+export const AdminDashboardLink = ({ link, text }) => (
```
Comment (radubuciuceanu) [inline src/admin/admin-dashboard-card/link.jsx]: Would the name `Link` be more relevant here?

File: src/startup/routes.js [hunk 5]
```diff
@@ -46,6 +46,7 @@ export const routes = [
     path: routesPath.Admin.Index,
     Component: Admin,
   },
+
```
Comment (radubuciuceanu) [inline src/startup/routes.js]: Let's rollback this file as we didn't make any changes here.

**Review by radubuciuceanu — CHANGES_REQUESTED** [review]

**Review by radubuciuceanu — APPROVED** [review]
Sexy :heart_eyes: 
Nice job!

**Pull request merged**